### PR TITLE
Update IsVehicleSirenOn and GetVehicleDoorAngleRatio

### PR DESCRIPTION
**Pull request by Flohhhhh**
Expected this to return false or true, instead returns false or 1.

Tested with prints.

```lua
CreateThead(function()
    while true do
        local veh = GetVehiclePedIsIn(PlayerPedId())
        print("[Lights Check] " .. tostring(IsVehicleSirenOn(veh)))
        Wait(1000)
    end
end)
```

image

### CODE REVIEW

**Comment by Flohhhhh** [comment]
First PR let me know if I screwed something : D

**Comment by logan-mcgee** [comment]
i want to say from memory, that this is intended/normal behaviour for CfxLua, and unrelated to the native used, but i could be wrong

**Comment by AvarianKnight** [comment]
It is for compatibility iirc, as long as you don't do specific `== true` or `== false` it shouldn't matter anyways

File: VEHICLE/IsVehicleSirenOn.md [hunk 1]
```diff
@@ -7,9 +7,18 @@ ns: VEHICLE
...
 BOOL IS_VEHICLE_SIREN_ON(Vehicle vehicle);
 ```
-
+Returns whether the vehicle's lights and sirens are on.
 
 ## Parameters
 * **vehicle**: 
 
 ## Return value
+Returns false if siren and lights are disabled or 1 if they enabled.
```
Comment (Flohhhhh) [inline VEHICLE/IsVehicleSirenOn.md:16]: Typo 😭 
"if they enabled" > "if they're enabled"

Comment (logan-mcgee) [reply VEHICLE/IsVehicleSirenOn.md:16]: the return value shouldnt be 1 anyway, as mentioned its purely for compatability, its just a truthy value and used correctly the value doesnt matter, its just true/false

**Comment by Flohhhhh** [comment]
Corrected typo and change from ``1`` to ``true``
Thanks everyone for the advice.

**Pull request merged**